docs/platform-integration/essentials/secure-storage.md

```diff
@@ -32,130 +35,148 @@ You can choose to disable Auto Backup for your entire application by setting the
 </manifest>
 ```
 
-### Selective Backup
+### Selective backup
+
 Auto Backup can be configured to disable specific content from backing up. You can create a custom rule set to exclude `SecureStore` items from being backed up.
 
-1. Set the `android:fullBackupContent` attribute in your **AndroidManifest.xml**:
+01. Set the `android:fullBackupContent` attribute in your _AndroidManifest.xml_:
 
     ```xml
     <application ...
         android:fullBackupContent="@xml/auto_backup_rules">
     </application>
     ```
 
-2. Create a new XML file named **auto_backup_rules.xml** in the **Resources/xml** directory with the build action of **AndroidResource**. Then set the following content that includes all shared preferences except for `SecureStorage`:
+01. Create a new XML file named _auto_backup_rules.xml_ in the _Resources/xml_ directory with the build action of **AndroidResource**. Set the following content that includes all shared preferences except for `SecureStorage`:
 
     ```xml
     <?xml version="1.0" encoding="utf-8"?>
     <full-backup-content>
         <include domain="sharedpref" path="."/>
-        <exclude domain="sharedpref" path="${applicationId}.xamarinessentials.xml"/>
+        <exclude domain="sharedpref" path="${applicationId}.mauiessentials.xml"/>
     </full-backup-content>
     ```
 
 # [iOS](#tab/ios)
 
 When developing on the **iOS simulator**, enable the **Keychain** entitlement and add a keychain access group for the application's bundle identifier.
 
-Open the **Entitlements.plist** in the iOS project and find the **Keychain** entitlement and enable it. This will automatically add the application's identifier as a group.
+Open the _Entitlements.plist_ in the iOS project and find the **Keychain** entitlement and enable it. This will automatically add the application's identifier as a group.
```

Comment:
That plist file is now in single project, rather than an iOS project. So I'd just say to open the file in the project.